src/homepage/howTo/custom-listeners.html

```diff
@@ -0,0 +1,237 @@
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+<!-- Main -->
+<div id="main">
+
+    <!-- Introduction -->
+    <section id="intro" class="main special">
+        <div class="">
+            <div class="content align-left">
+                <header class="major">
+                    <h1><b>Configure Custom Listeners</b></h1>
+                </header>
+
+                <p>
+                    This document will help you understand what is Mailbox Listener, then have a chance to implement your own Mailbox Listener
+                    and configure to use it in James server.
+                </p>
+
+                <ul>
+                    <li>What is Mailbox Listener</li>
+                    <li>How to use a custom Mailbox Listener in James</li>
+                </ul>
+
+                <header class="major">
+                    <h2><b>What is Mailbox Listener</b></h2>
+                </header>
+
+                <p>
+                    Mailbox Listener is a component in James Mailbox System.
+                    Every time, if there is an action applied on a mailbox(adding, deleting), or an email(adding, deleting, updating flags...).
+                    Then an event represented for that action is generated and delivered to all Listeners that have registered before.
+                    After receiving events, listeners retrieve information from the events then execute their business
+                    (Indexing emails in ElasticSearch, updating quota of users, detecting spam emails...)
+                </p>
+
+                <p>
+                    There are two kinds of Listener registration:
+                </p>
+                <ul>
+                    <li>
+                        <b>Group Registration</b>: If a Listener is registered as the Group Registration, it will receive all of events fired by James Mailbox System.
+                        You can write your own custom Listener and configure James to register it as a Group Registration.
+                    </li>
+                    <li>
+                        <b>Key Registration</b>: If a Listener is registered as the Key Registration, it will receive only events it is interested in.
+                        At the moment, James doesn't support custom listeners Key Registration
+                    </li>
+                </ul>
+
+                <header class="major">
+                    <h2><b>How to use a custom Mailbox Listener in James</b></h2>
+                </header>
+
+                <p>
+                    <b>prerequisite</b>: custom Mailbox Listeners can only work with James Guice products.
+                </p>
+
+                <header class="major">
+                    <h3><b>Use the custom BigMessageListener in James</b></h3>
+                </header>
+
+                <p>
+                    Once you have a custom Listener, setup James with that Listener is very simple.
```

Comment:
setup James with that Listener is very simple.
=>
It's very simple to setup James with that Listener.